src/box/box.cc

```diff
@@ -3635,9 +3693,13 @@ local_recovery(const struct tt_uuid *instance_uuid,
 		box_update_replication();
 
 		struct replica *master;
-		if (replicaset_needs_rejoin(&master)) {
+		while (replicaset_needs_rejoin(&master)) {
 			say_crit("replica is too old, initiating rebootstrap");
```

Comment:
This log doesn't need to be repeated. Would be better to log it out of the loop, above.

Reply:
Done